chromadb/proto/logservice_pb2_grpc.py

```diff
@@ -15,10 +15,15 @@ def __init__(self, channel):
             channel: A grpc.Channel.
         """
         self.PushLogs = channel.unary_unary(
-                '/chroma.LogService/PushLogs',
-                request_serializer=chromadb_dot_proto_dot_logservice__pb2.PushLogsRequest.SerializeToString,
-                response_deserializer=chromadb_dot_proto_dot_logservice__pb2.PushLogsResponse.FromString,
-                )
+            "/chroma.LogService/PushLogs",
+            request_serializer=chromadb_dot_proto_dot_logservice__pb2.PushLogsRequest.SerializeToString,
+            response_deserializer=chromadb_dot_proto_dot_logservice__pb2.PushLogsResponse.FromString,
+        )
```

Comment:
Would be nice to add gitattributes to not see generated files in pr.

Reply:
sure!